akka-projection-core/src/main/scala/akka/projection/ProjectionBehavior.scala

```diff
@@ -139,9 +144,28 @@ object ProjectionBehavior {
         context.log.warn2("Operation [{}] failed with: {}", op, exc)
         Behaviors.same
 
+      case setPaused: SetPaused =>
+        running match {
+          case mgmt: RunningProjectionManagement[_] =>
+            if (setPaused.projectionId == projectionId) {
+              context.log.infoN(
+                "Paused will be changed to [{}] for projection [{}]. The Projection processing will be {}.",
```

Comment:
I find this logging weird. What about the following...

"Running status will be changed to [paused|resumed] for projection [some-id].",